[2024-04-25T07:57:16.042+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T07:57:16.068+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2023-03-14T00:00:00+00:00 [queued]>
[2024-04-25T07:57:16.080+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2023-03-14T00:00:00+00:00 [queued]>
[2024-04-25T07:57:16.085+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-25T07:57:16.101+0000] {taskinstance.py:2327} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2023-03-14 00:00:00+00:00
[2024-04-25T07:57:16.109+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3496) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T07:57:16.114+0000] {standard_task_runner.py:63} INFO - Started process 3514 to run task
[2024-04-25T07:57:16.109+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'insert_into_table', 'scheduled__2023-03-14T00:00:00+00:00', '--job-id', '4061', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpo9zutveo']
[2024-04-25T07:57:16.115+0000] {standard_task_runner.py:91} INFO - Job 4061: Subtask insert_into_table
[2024-04-25T07:57:16.173+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.insert_into_table scheduled__2023-03-14T00:00:00+00:00 [running]> on host 4d994982806b
[2024-04-25T07:57:16.315+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2023-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-03-14T00:00:00+00:00'
[2024-04-25T07:57:16.321+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T07:57:16.341+0000] {sql.py:276} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2023-03-14', 'dag_with_postgres_operator_v01')
        
[2024-04-25T07:57:16.366+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-25T07:57:16.379+0000] {base.py:84} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-04-25T07:57:16.408+0000] {sql.py:457} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2023-03-14', 'dag_with_postgres_operator_v01')
        , parameters: None
[2024-04-25T07:57:16.423+0000] {sql.py:466} INFO - Rows affected: 1
[2024-04-25T07:57:16.429+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/common/sql/hooks/sql.py:407 AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
[2024-04-25T07:57:16.445+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T07:57:16.459+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=insert_into_table, execution_date=20230314T000000, start_date=20240425T075716, end_date=20240425T075716
[2024-04-25T07:57:16.554+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T07:57:16.580+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T07:57:16.587+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
